Supported migration network for MTV (#578)
kubev2v/forklift-documentation: documentation/modules/creating-migration-plan-2-6-3.adoc

+
If the conversion fails because the boot device provided is incorrect, it is possible to get the correct information by looking at the conversion pod logs.

> * *{ocp-short} transfer network*: You can configure an {ocp-short} network in the ocp-short} web console by clicking *Networking > NetworkAttachmentDefinitions*.

Reviewer: can we tidy up the naming convention, please?
it does not look good on URLs for MTV 2.7 to see 2.6.3 in the URL

Author: Good point. Fixed.

> * *{ocp-short} transfer network*: You can configure an {ocp-short} network in the ocp-short} web console by clicking *Networking > NetworkAttachmentDefinitions*.

Reviewer: ```suggestion
* *{ocp-short} transfer network*: You can configure an {ocp-short} network in the {ocp-short} web console by clicking *Networking > NetworkAttachmentDefinitions*.
```
Check your previews to ensure the attributes render correctly

Author: Fixed in creating-migration-plan-2.7.adoc.

+

> To learn more about the different types of networks {ocp-short} supports, see https://docs.openshift.com/container-platform/4.16/networking/multiple_networks/understanding-multiple-networks.html#additional-networks-provided [Additional Networks In OpenShift Container Program].

Reviewer: ```suggestion
To learn more about the different types of networks {ocp-short} supports, see https://docs.openshift.com/container-platform/{ocp-version}/networking/multiple_networks/understanding-multiple-networks.html#additional-networks-provided [Additional Networks in OpenShift Container Program].
```
Let us track the latest OCP version
I need to add a `404` check to the repo

Author: Fixed in creating-migration-plan-2.7.adoc.

+

> If you want to change the MTU of the {ocp-short} transfer network, you must also change the MTU of the VMware migration network. For more information see xref:selecting-migration-network-for-vmware-source-provider_vmware[Selecting a migration network for a VMware source provider].

Reviewer: ```suggestion
If you want to adjust the maximum transmission unit (MTU)  of the {ocp-short} transfer network, you must also change the MTU of the VMware migration network. For more information see xref:selecting-migration-network-for-vmware-source-provider_vmware[Selecting a migration network for a VMware source provider].
```

> If you want to change the MTU of the {ocp-short} transfer network, you must also change the MTU of the VMware migration network. For more information see xref:selecting-migration-network-for-vmware-source-provider_vmware[Selecting a migration network for a VMware source provider].

Reviewer: ```suggestion
If you want to change the MTU of the {ocp-short} transfer network, you must also change the MTU of the VMware migration network. For more information, see xref:selecting-migration-network-for-vmware-source-provider_vmware[Selecting a migration network for a VMware source provider].
```
